.gitignore

```diff
@@ -2,4 +2,5 @@ node_modules/
 .nyc_output/
 .DS_Store
 package-lock.json
+yarn.lock
```

Comment:
yarn really needs the lockfile unfortunately :(. I'm also ok in ahving the lockfile for trainings.

Reply:
Why do you think it needs it?